proc_square_refl.py

```diff
@@ -143,19 +143,28 @@ def filter_range(x): return array([j for j in x if
     d.setaxis('t', lambda x: x-time_zero).register_axis({'t':0})
     refl_blip_ranges -= time_zero
     pulse_range -= time_zero
-    fl.next('after setting t=0')
-    for j in range(2):
-        fl.plot(abs(d['ch',j]), linewidth=3, color='k',
-                label='ch %d abs'%(j+1), alpha=0.3)
     d = d['t':(-10e6,10e6)] # slice out frequencies with signal
     #}}}
     
     #{{{zeroth order phase correction
     for j in range(2):
+        fl.basename = "channel %d"%(j+1)
         ph0 = zeroth_order_ph(d['ch',j], fl=fl)
         d['ch',j] /= ph0
+    fl.basename = None
     #}}}
```

Comment:
puts the zeroth order diagnostic plots on separate plots -- this is a good test case for issue #23  -- we can see from the diagnostic plots that it's not functioning correctly.  You just need to add weights when calling `cov`